test/test-runner-commands/index.js

```diff
@@ -2,6 +2,41 @@ import { executeServerCommand } from '@web/test-runner-commands';
 
 export * from '@web/test-runner-commands';
 
+/**
+ * Moves the mouse to the center of an element and optionally clicks
+ * a mouse button on it.
+ *
+ * After using this function, remember to call `resetMouse` to reset
+ * the mouse position and avoid affecting other tests.
+ *
+ * @typedef {{ type: 'move', element: Element }} MovePayload
+ * @typedef {{ type: 'click', element: Element, button?: 'left' | 'middle' | 'right' }} ClickPayload
+ * @param {Element | MovePayload | ClickPayload} payload
+ * @return {Promise<void>}
+ *
+ * @example
+ * // Move the mouse to the center of an element
+ * await sendMouseToElement(document.querySelector('#my-element'));
+ *
+ * @example
+ * // Click the left mouse button on an element
+ * await sendMouseToElement({ type: 'click', element: document.querySelector('#my-element') });
+ */
+export async function sendMouseToElement(payload) {
+  let element;
+  if (payload instanceof Element) {
```

Comment:
I feel using the element as the payload isn't a large improvement and is somewhat confusing as `sendMouseToElement(element)` doesn't tell you what it does. So I'd prefer the more explicit form with type and element.

Reply:
Thanks, done.